src/Tribe/Template.php

```diff
@@ -501,7 +501,7 @@ public function get_template_file( $name ) {
 	 *
 	 * @return string|false Either the final content HTML or `false` if no template could be found.
 	 */
-	public function template( $name, $context = array(), $echo = true ) {
+	public function template( $name, $context = [], $echo = true ) {
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 592c64993667a33767f4f7e5a120dac0</sub>